public/docs/content/getting_started/index.md

```diff
@@ -0,0 +1,2 @@
+# Getting started
+
```

Comment:
This is empty and can be removed.

Reply:
comment: I'd like to add summary content here